### PR TITLE
Deploy the emojis app on every push.

### PR DESCRIPTION
**Pull request by spetrovic77**

### CODE REVIEW

File: 09/config.toml [hunk 1]
```diff
@@ -6,3 +6,11 @@ listeners.emojis = {address = "localhost:9000"}
...
 [multi]
 listeners.emojis = {address = "localhost:9000"}
+
+[kube]
+listeners.emojis = {public = true}
+image = "us-central1-docker.pkg.dev/serviceweaver-394516/github-deploy/emojis"
+
+[gke]
+regions = ["us-west1"]
+listeners.emojis = {public_hostname = "emojis.serviceweaver.dev" }
```
Comment (rgrandl) [inline 09/config.toml:16]: we don't need this no?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        )
+        echo "$CONFIG" > ${{ env.CONFIG_FILE }}
+        cat ${{ env.CONFIG_FILE }}
+
+    - name: Build the YAML files
+      run: weaver-kube deploy ${{ env.CONFIG_FILE }}
+
+    - name: Deploy the YAML files
+      run: |
+        FILE=$(ls *.yaml)
```
Comment (rgrandl) [inline .github/workflows/deploy.yml:91]: Is this guaranteed to be a single yaml file?

**Review by rgrandl — APPROVED** [review]
Looks very elegant!

File: 09/config.toml (hunk 1 above)
Comment (spetrovic77) [reply 09/config.toml:16]: Good point, deleted.

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (spetrovic77) [reply .github/workflows/deploy.yml:91]: Yes, because we clone everything and run once.

**Pull request merged**